openbb_platform/providers/federal_reserve/openbb_federal_reserve/models/money_measures.py

```diff
@@ -0,0 +1,103 @@
+"""FederalReserve Money Measures Model."""
+
+from datetime import datetime, timedelta
+from io import BytesIO
+from typing import Any, Dict, List, Optional
+
+import pandas as pd
+from numpy import nan
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.money_measures import (
+    MoneyMeasuresData,
+    MoneyMeasuresQueryParams,
+)
+from openbb_core.provider.utils.helpers import make_request
+from pydantic import field_validator
+
+titles = {
+    "M1": "M1",
```

Comment:
To be consistent with everything else, we should probably make this lower_snake_case.  M1 and M2 could stay like that, IMO, dealer's choice.